session.go

```diff
@@ -82,6 +82,12 @@ type Session interface {
 	// the request handling loop. Registering nil will unregister the channel.
 	// During the time that no channel is registered, breaks are ignored.
 	Break(c chan<- bool)
+
+	// Stderr returns an io.ReadWriter that writes to this channel
+	// with the extended data type set to stderr. Stderr may
+	// safely be read and written from a different goroutine than
+	// Read and Write respectively.
+	Stderr() io.ReadWriter
```

Comment:
Shouldn't this be a plain io.Writer? I'm fairly certain you can't read data from stderr.

Reply:
IMHO it could, but it comes as a readwriter from ssh.Channel: https://pkg.go.dev/golang.org/x/crypto/ssh#Channel

which we embed on our session here: https://github.com/gliderlabs/ssh/blob/db094653c80809f63e06c08eda6987780c3c4a08/session.go#L23

That said, if I try to change it to writer only, it won't compile... so unfortunately I think there's no other way (besides maybe a new stderr method with another name).